docs/platform/howto/cleanup-powered-off-services.rst

```diff
@@ -0,0 +1,22 @@
+Periodic cleanup of powered-off services
+========================================
+
+Aiven platform provides a way to power-off services stop accumulating usage costs when not in use. However leaving them powered-off for an extended period of time comes with disadvantages too.
+
+Aiven platform receive regular maintenance updates to keep the services updated with the latest supported versions upstream, including fixes to bugs, possible security vulnerabilities, and overall improvements to performance and memory consumption.
+
+Keeping services in powered-off state for a long time lowers the feasibility of a smooth upgrade path, making it harder for us to continue supporting your needs.
+
+.. note:: 
+    * It is recommended that you regularly review your services and delete those that are no longer needed. This will allow Aiven to focus on supporting the services that you actively use and better utilize platform resources. 
```

Comment:
1. replace Future with Present for generalizing over this idea:

This allows Aiven to focus on supporting the services that you actively use and better utilize platform resources.